genome_sampler/label_seqs.py

```diff
@@ -4,13 +4,19 @@
 
 
 def label_seqs(seqs: pd.Series, delimiter: str,
-               metadata: qiime2.Metadata = None, columns: str = None) \
+               metadata: qiime2.Metadata = None, columns: str = None,
+               missing_value: str = 'missing') \
                    -> pd.Series:
     if columns is not None and metadata is None \
             or metadata is not None and columns is None:
         raise ValueError('Columns and metadata must be passed or not passed '
                          'together.')
 
+    if delimiter in missing_value:
+        raise ValueError(f'The given delimiter {repr(delimiter)} is contained '
+                         'within your missing value placeholer '
```

Comment:
@Oddant1, I think the text I suggested here is a clearer error message. Just wanted to make sure you saw that suggestion since there were two suggestions in this one comment. If you disagree though we can leave this error message as-is, I don't have strong feelings about this change.

Reply:
Oh, I didn't see it. Thank you.